modules/core/src/main/java/org/apache/fluo/core/thrift/Stamps.java

```diff
@@ -21,6 +21,7 @@
 import org.apache.thrift.scheme.StandardScheme;
 import org.apache.thrift.scheme.TupleScheme;
 
+@SuppressWarnings({"rawtypes", "unused", "serial", "unchecked"})
```

Comment:
this class is generated by thrift. is this ok to add?  will it be lost if generated again?